engine.go

```diff
@@ -102,15 +114,35 @@ func (e *Engine) applyRemove(sec string, ptype string, rule []string) (bool, err
 
 // getSnapshot convert model data to snapshot
 func (e *Engine) getSnapshot() ([]byte, error) {
-	return json.Marshal(e.enforcer.GetModel())
```

Comment:
Why don't we use json struct?

Reply:
json can't unmarshal interface, so an error occurs when parsing Rolemanager

And using the entire model comes with the information in model.conf, which is useless, we only need the policy